docs/pages/deploy-a-cluster/aws-gslb-proxy-peering-ha-deployment.mdx

```diff
@@ -0,0 +1,252 @@
+- [AWS Route 53 latency based routing](https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/routing-policy-latency.html) 
+- [GSLB](https://www.cloudflare.com/learning/cdn/glossary/global-server-load-balancing-gslb/)
+- [Teleport TLS Routing](https://goteleport.com/docs/architecture/tls-routing/) to reduce the number of ports needed to use Teleport
+- [Teleport Proxy Peering](https://goteleport.com/docs/architecture/proxy-peering/) for reducing the number of resource connections
+- [AWS Network Load Balancing](https://docs.aws.amazon.com/elasticloadbalancing/latest/network/introduction.html)
+- [AWS DynamoDB](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Introduction.html) for cluster state storage
+- [AWS S3](https://docs.aws.amazon.com/AmazonS3/latest/userguide/Welcome.html) for session recording storage
+
+## Advantages of this deployment architecture
+- Eliminates the complexity maintaining multiple Teleport clusters across multiple regions
+- Uses the lowest latency path to connect users to resources
+- Provides a highly-resilient, redundant HA architecture for Teleport that can quickly 
+  scale with an organization’s needs.
+- All required Teleport components can be provisioned within the AWS ecosystem.
+- Using load balancers for the Proxy and Auth services allows for increased availability 
+  during Teleport Cluster upgrades.
+
+## Disadvantages of this deployment architecture
+- When Teleport Auth servers are limited to a single region, there is a higher likelihood 
+  of decreased availability during an AWS regional outage.
+- Technically complex to deploy
+
+![Diagram of a high-availability Teleport
+architecture](../../img/deploy-a-cluster/aws-gslb-proxy-peering-ha-deployment.png)
+
+
+## AWS Network load balancer (NLB)
+AWS NLBs are required for this highly available deployment architecture. 
+The NLB forwards traffic from users and services to an available Teleport instance. This must not 
+terminate TLS, and must transparently forward the TCP traffic it receives. 
+In other words, this must be a Layer 4 load balancer, not a Layer 7 
+(e.g., HTTP) load balancer. 
+
+<Admonition
+  type="warning"
+  title="Note"
+>
+Cross-zone load balancing is requried for the Auth and Proxy service NLB configurations to route 
+traffic across multiple zones. Doing this improves resiliency against localized AWS zone outages.
+</Admonition>
+
+### Configure the Proxy Service NLBs
+Configure the load balancer to forward traffic from the following ports on the
+load balancer to the corresponding port on an available Teleport instance. The
+configuration depends on whether you route Proxy Peering traffic over 
+the public internet:
+
+<Tabs>
+<TabItem label="Public NLB Proxy ports">
+
+| Port | Description |
+| - | - |
+| `443` | ALPN port for TLS Routing,  HTTPS connections to authenticate `tsh` users into the cluster, and to serve a Web UI |
+| `3021`| Proxy Peering gRPC Stream  |
```

Comment:
@webvictim Instead of removing this suggestion, should we reframe it as something that is technically possible that we advise against? I'm going to guess that there will be >0 people that try to set it up this way. Or we could add on a troubleshooting section and include this as a possible misconfiguration?